packages/devtools/src/scripts/executeScript.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * injector script for executeScript command
+ * According to the WebDriver spec we need to ensure to serialize all elements
+ * being returned from the script. In order to do so we attach a data attribute
+ * to them and fetch them using Puppeteer after the script execution. Instead
+ * of the element we return the dataFlag string instead to signalise that this
+ * is suppose to be an element handle
+ *
+ * @param  {HTMLElement} _            $eval fetched element
+ * @param  {String}      script       user script
+ * @param  {String}      dataProperty property name for elements being returned
+ * @param  {String}      dataFlag     flag for element
+ * @param  {Object[]}    args         user arguments for custom script
+ * @return {Object}                   result of custom script
+ */
+export default function executeScript (_, script, dataProperty, dataFlag, ...args) {
```

Comment:
this won't accept element as argument, it changes it to body if function is not arrow function.

It should be changed to same way as executeAsyncScript is designed.

`export default (_, script, dataProperty, dataFlag, ...args) => {`